src/Modal/Modal.js

```diff
@@ -98,8 +105,30 @@ function Modal({ children, title, animationDuration, showClose, onClose, open, .
     if (open !== isOpen) {
       setOpen(open);
     }
+
+    if (blur && blurNode) {
+      toggleBlur();
+    }
   }, [open]);
 
+  useEffect(() => {
+    if (blur) {
+      const target = document.getElementById(blurTarget);
```

Comment:
Maybe we should use `document.querySelector` to provide a more flexible API? e.g someone might want to use a class name, etc

Reply:
True. That would be more flexible.